pkg/odo/cli/catalog/describe/service.go

```diff
@@ -42,20 +62,88 @@ func NewDescribeServiceOptions() *DescribeServiceOptions {
 
 // Complete completes DescribeServiceOptions after they've been created
 func (o *DescribeServiceOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	isSVCSupported, err := svc.IsCSVSupported()
+	if err != nil {
+		isSVCSupported = false
```

Comment:
This is wrong, IMO. If there's an error, there's something that went wrong. It should not be interpreted as CSV support being inexistent. Also, it's CSV, not SVC. CSV is for `ClusterServiceVersion`.

Reply:
For example, 
* We ignore the error here since we don't care: https://github.com/openshift/odo/blob/3b11f9d5612f9ee26494748ad8fa926d7b79fbd6/pkg/odo/cli/component/link.go#L97
* It's treated separately here: https://github.com/openshift/odo/blob/3b11f9d5612f9ee26494748ad8fa926d7b79fbd6/pkg/odo/cli/component/unlink.go#L63-L73
* And here we do different things based on different possibilities. https://github.com/openshift/odo/blob/3b11f9d5612f9ee26494748ad8fa926d7b79fbd6/pkg/odo/cli/service/list.go#L47-L53

Reply:
Please recommend an action that you feel is correct if there is an error while checking for csv? Should we error out as the format matches?

Reply:
changed to approach. and added comments, please check if it makes sense.

Reply:
> Please recommend an action that you feel is correct if there is an error while checking for csv? 

If there's an error, we should throw the error since it's an error.

Reply:
done